### DISCUSSION TITLE
Install not working

### DISCUSSION THREAD

**Discussion by jamiescott77** [aws-samples/retail-demo-store#510] [Q&A]
Hi Everyone

I've installed AWS Personalize Retail Store twice now and each time get the same result as in No Products are loading 
<img width="1100" alt="Screenshot 2023-11-05 at 15 15 16" src="https://github.com/aws-samples/retail-demo-store/assets/149938139/aabdff76-8141-4f42-b3a4-0e2ccf2fc326">

Any ideas?

**Comment by BastLeblanc** [comment]
closing as it's an old issue. Please re-open if you still encounter the issue
